docs/book/v2/quick-start.md

```diff
@@ -0,0 +1,161 @@
+# Quick Start
+
+## Installation
+
+Use composer to install the library:
+
+```bash
+$ composer require laminas/laminas-mvc-middleware
+```
+
+If the [laminas-component-installer](https://docs.laminas.dev/laminas-component-installer/) is installed in your project
+it will register the new component automatically. Otherwise, you have to add `Laminas\Mvc\Middleware` at the **top** of your
+`config/modules.config.php` file.
```

Comment:
Can be removed because we have already an installation page with content. To activate this page:

1. [add an entry to the navigation](https://github.com/laminas/laminas-i18n/blob/99519a72c3a38ab0522df332a732841517c16332/mkdocs.yml#L6) in MkDocs configuration
2. [add the module class name](https://github.com/laminas/laminas-i18n/blob/99519a72c3a38ab0522df332a732841517c16332/mkdocs.yml#L50-L52) in MkDocs configuration
3. [create an empty file](https://github.com/laminas/laminas-i18n/blob/2.12.x/docs/book/installation.md) for the page